apps/fluent-tester/src/TestComponents/Shadow/ShadowWithDifferentPropsTestSection.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { Text } from '@fluentui/react-native';
+import { Shadow } from '@fluentui-react-native/experimental-shadow';
+import { Theme, useFluentTheme } from '@fluentui-react-native/framework';
+import { View } from 'react-native';
+import { themedStyleSheet } from '@fluentui-react-native/themed-stylesheet';
+import { mergeStyles } from '@fluentui-react-native/merge-props';
+
+const getThemedStyles = themedStyleSheet((t: Theme) => {
+  return {
+    defaultShadowTestBoxProps: {
+      maxWidth: 732,
+      minHeight: 64,
+      padding: 20,
+      marginVertical: 16,
+      marginHorizontal: 32,
+      backgroundColor: t.colors.background,
+    },
+  };
+});
+
+export const ShadowWithDifferentPropsTestSection: React.FunctionComponent = () => {
+  const theme = useFluentTheme();
+  const themedStyles = getThemedStyles(theme);
+
+  return (
+    <View>
+      <Text style={{ padding: 16 }}>The following tests have a Shadow16 set on a child view that has some specific props.</Text>
```

Comment:
Nit: inline style

Reply:
Moved to the themed stylesheet. Also moved some stuff that was in the style objects on the other tests below into the style sheet